pkg/build/builder/util.go

```diff
@@ -1,7 +1,11 @@
 package builder
 
 import (
-	"github.com/openshift/origin/pkg/build/api"
+	"github.com/golang/glog"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	buildutil "github.com/openshift/origin/pkg/build/util"
+	osclient "github.com/openshift/origin/pkg/client"
```

Comment:
clarity.  in some places we have the k8s client too, and i don't want to have to wonder which this is when looking at the code later, and have to jump up to the imports to get clarity.